src/MSBuild.Abstractions/MsBuildExtensions.cs

```diff
@@ -8,11 +8,11 @@ namespace MSBuild.Abstractions
 {
     public static class MsBuildExtensions
     {
-        public static XmlElement GetXml(this ProjectItemElement projectItem)
+        public static XmlElement? GetXml(this ProjectItemElement projectItem)
         {
-            return (XmlElement)GetPropertyValue(projectItem, "XmlElement");
+            return GetPropertyValue(projectItem, "XmlElement") as XmlElement;
```

Comment:
When will this not be an `XmlElement`? Why not just explicitly cast it?

Reply:
This was a result of marking `GetPropertyValue `as nullable.
I'll revert that.